game/gbtd_gbr0.ksy

```diff
@@ -0,0 +1,300 @@
+            1: top2bottom_left2right
+
+          color_set:
+            0: pocket
+            1: gameboy
+            2: gbc
+            3: sgb
+
+        types:
+          display_settings:
+            seq:
+              - id: grid_shown
+                type: b1
+              - id: nibble_markers_shown
+                type: b1
+
+        seq:
+          - id: tile_id
+            type: u2
+          - id: simple
+            type: u1
+          - id: flags
+            type: display_settings
+          - id: left_color
+            type: u1
+          - id: right_color
+            type: u1
+          - id: split_width
+            type: u2
+          - id: split_height
+            type: u2
+          - id: split_order
+            type: u1 # Note: Official specification incorrectly says "long"
+            enum: split_order
+          - id: color_set
+            type: u1
+            enum: color_set
+          - id: bookmarks
+            type: u2
+            repeat: expr
+            repeat-expr: 3
+          - id: auto_update
+            type: u1
+        
+      tile_export:
+        enums:
+          compression:
+            0x00: none
+            0x01: gb_compress
+
+          format:
+            0x00: gameboy_4_color
+            0x01: gameboy_2_color
+            0x03: byte_per_color
+
+          counter_type:
+            0x00: none
+            0x01: count_byte_as_byte
+            0x02: count_byte_as_word
+            0x03: count_byte_as_constant
+            0x04: count_tile_as_byte
+            0x05: count_tile_as_word
+            0x06: count_tile_as_constant
+            0x07: count_8x8_as_byte
+            0x08: count_8x8_as_word
+            0x09: count_8x8_as_constant
+
+          palettes:
+            0: none
+            1: constant_per_entry
+            2: bits_2_per_entry
+            3: bits_4_per_entry
+            4: byte_1_per_entry
+
+        seq:
+          - id: tile_id
+            type: u2
+          - id: file_name
+            type: strz
+            size: 128
+          - id: file_type
+            type: u1
+          - id: section_name
+            type: strz
+            size: 20
+          - id: label_name
+            type: strz
+            size: 20
+          - id: bank
+            type: u1
+          - id: tile_array
+            type: u1
+          - id: format
+            type: u1
+            enum: format
+          - id: counter
+            type: u1
+            enum: counter_type
+          - id: export_from
+            type: u2
+          - id: export_to
+            type: u2
+          - id: compression
+            type: u1
+            enum: compression
+          - id: include_colors
+            type: u1
+          - id: sgb_palettes
+            type: u1
+            enum: palettes
+          - id: gbc_palettes
+            type: u1
+            enum: palettes
+          - id: make_meta_tiles
+            type: u1
+          - id: meta_offset
```

Comment:
Is it an offset of some structure that should be a `pos` `instance`?

Reply:
Possibly. It sounds like it is a offset/type but it's not described in the spec and I don't know exactly how it works.

Reply:
Could you, please, create a `doc:` with this information?

Reply:
Added a small doc comment about this. Will try to dig deeper at a later point to figure out this structure. It probably isn't very difficult.